instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/logs/LoggerE2ETests.kt

```diff
@@ -250,4 +264,245 @@ class LoggerE2ETests {
             )
         }
     }
+
+    /**
+     * apiMethodSignature: Logger#fun addAttribute(String, Boolean)
+     */
+    @Test
+    fun logs_logger_add_boolean_attribute() {
+        val testMethodName = "logs_logger_add_boolean_attribute"
+        measure(testMethodName) {
+            logger.addAttribute(SPECIAL_BOOL_ATTRIBUTE_NAME, forge.aBool())
+        }
+        logger.sendRandomLog(testMethodName, forge)
+    }
+
+    /**
+     * apiMethodSignature: Logger#fun addAttribute(String, Int)
+     */
+    @Test
+    fun logs_logger_add_int_attribute() {
+        val testMethodName = "logs_logger_add_int_attribute"
+        measure(testMethodName) {
+            logger.addAttribute(SPECIAL_INT_ATTRIBUTE_NAME, CUSTOM_INT_ATTRIBUTE)
+        }
+        logger.sendRandomLog(testMethodName, forge)
+    }
+
+    /**
+     * apiMethodSignature: Logger#fun addAttribute(String, String?)
+     */
+    @Test
+    fun logs_logger_add_string_attribute() {
+        val testMethodName = "logs_logger_add_string_attribute"
+        measure(testMethodName) {
+            logger.addAttribute(SPECIAL_STRING_ATTRIBUTE_NAME, CUSTOM_STRING_ATTRIBUTE)
```

Comment:
You can have some diversity if you look for it with a wildcard : `value:foo*` will match both `foo` and `food`

Reply:
that's nice, I didn't know we could use the wildcard like that in DD